Sources/BarcodeScannerController.swift

```diff
@@ -343,8 +343,6 @@ open class BarcodeScannerController: UIViewController {
         center(subview: focusView, inSize: CGSize(width: 218, height: 150))
     }
     center(subview: settingsButton, inSize: CGSize(width: 150, height: 50))
-
-    headerView.isHidden = !isBeingPresented
```

Comment:
@chadcummings What is the reason behind this change?

Reply:
This fixes a bug when the screen orientation was changed, the Header would disappear and never reappear, even when the device was rotated back. I haven't noticed any negative side effects, but I'll look to you for guidance here.

Reply:
I think it was here for the case when you have `UINavigationController` and push barcode scanner controller, so you don't end up with two "navigation bars". But arguably it's something you can manually control in the code.

Reply:
I'll see if I can test this out by enhancing the example app to verify. I'll fix it if necessary.

Reply:
@chadcummings Keep us posted. It's the only thing we'd like to verify before merging this PR.